### PR TITLE
__init__.py: Arrange imports alphabetically

### LINKED ISSUE #5745: Imported classes should be arranged lexicographically
Classes being imported in `coala/coalib/bearlib/languages/__init__.py` file should be arranged alphabetically.

A good newcomer issue.

### PR DESCRIPTION
**Pull request by subhaminion**
Fixes  #5745 
I've used a tool called `isort` https://pypi.org/project/isort/.

It helps sorting the imported module in alphabetical order just by running a command and saves time.

### CODE REVIEW

File: coalib/bearlib/languages/__init__.py [hunk 1]
```diff
@@ -17,26 +12,30 @@
...
 from .definitions.Tcl import Tcl
-from .definitions.Vala import Vala
 from .definitions.TypeScript import TypeScript
-from .definitions.Shell import Shell
-from .definitions.Jinja2 import Jinja2
+from .definitions.Unknown import Unknown
+from .definitions.Vala import Vala
 from .definitions.VisualBasic import VisualBasic
 from .definitions.XML import XML
+# Start ignoring PyUnusedCodeBear
```
Comment (sangamcse) [inline coalib/bearlib/languages/__init__.py]: No need to change this line

File: coalib/bearlib/languages/__init__.py [hunk 2]
```diff
@@ -17,26 +12,30 @@
...
+from .Language import Language, Languages
```
Comment (sangamcse) [inline coalib/bearlib/languages/__init__.py]: Break these 2 imports as it was earlier

Comment (sangamcse) [reply coalib/bearlib/languages/__init__.py]: Also, add a blank line before `Language` import.

**Comment by ManthanKeim** [comment]
There's a redundant space in your commit message. Better if you fix it.

**Comment by subhaminion** [comment]
@sangamcse Fixed check once.

File: coalib/bearlib/languages/__init__.py [hunk 3]
```diff
@@ -17,26 +13,31 @@
...
-from .definitions.Vala import Vala
 from .definitions.TypeScript import TypeScript
-from .definitions.Shell import Shell
-from .definitions.Jinja2 import Jinja2
+from .definitions.Unknown import Unknown
+from .definitions.Vala import Vala
 from .definitions.VisualBasic import VisualBasic
 from .definitions.XML import XML
+
+from .Language import Language
```
Comment (sangamcse) [inline coalib/bearlib/languages/__init__.py]: Don't you think it needs to be below `from.definitions.ZShell import ZShell`?
Also, fix `from.definitions.ZShell import ZShell` to `from .definitions.ZShell import ZShell`.

**Comment by subhaminion** [comment]
@sangamcse done

**Comment by sangamcse** [comment]
Your commit is here https://github.com/coala/coala/pull/5814/commits, not https://github.com/coala/coala/pull/5814#issue-222699122

**Comment by jayvdb** [comment]
Read https://coala.io/commit carefully.

**Comment by subhaminion** [comment]
@jayvdb see now.
If it doesn't complies then I need help. :D

**Comment by subhaminion** [comment]
@jayvdb Hey can you check once and merge?

File: coalib/bearlib/languages/__init__.py (hunk 3 above)
Comment (jayvdb) [reply coalib/bearlib/languages/__init__.py]: Sorry, only the long list of languages should be re-ordered.

Each of the languages is a `Language` instance.

Therefore loading `Language` should occur before trying to load an instance of `Language`.

Comment (subhaminion) [reply coalib/bearlib/languages/__init__.py]: Hey did't get that can you elaborate @jayvdb

Comment (chay2199) [reply coalib/bearlib/languages/__init__.py]: The following imports
```
from .Language import Language
from .Language import Languages
```
should not be re-ordered and only the list of instances of languages should.

**Comment by jayvdb** [comment]
Needs rebase.  using `git`.

File: coalib/bearlib/languages/__init__.py [hunk 4]
```diff
@@ -6,7 +6,6 @@
 from .Language import Language
 from .Language import Languages
 
-from .definitions.Unknown import Unknown
```
Comment (jayvdb) [inline coalib/bearlib/languages/__init__.py]: unknown is intentionally first.  it isnt a language.

Add a blank line under it to make this more obvious.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5814/commits/eaacdcd49a3a3cc00e4cbffe148b283e8055a430

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**